packages/asset-swapper/src/network/samplers/balancer.ts

```diff
@@ -0,0 +1,250 @@
+import { ChainId } from '@0x/contract-addresses';
```

Comment:
Balancer is an example of a source that that relies on off-chain queries and caching, but now it's neatly scoped to the balancer integration so no else one has to know or worry about it.